aiohttp/client.py

```diff
@@ -41,7 +42,7 @@
 
 
 # 5 Minute default read and connect timeout
-DEFAULT_TIMEOUT = 5 * 60
+DEFAULT_TIMEOUTS = RequestTimeouts(uber_timeout=5 * 60)
```

Comment:
sounds like this should be request_timeout instead of uber

Reply:
I thought the old timeout worked like the uber_timeout is working now, so I though this would be suited best for keeping backwards compatibility. If changing the way things work is okay, I'd be happy to use a more sensible timeout for the default. ;)